### PR TITLE
Updated geoviews to use param.Version

### PR DESCRIPTION
**Pull request by jlstevens**
Follows how holoviews does it and allows ``version.py`` to be removed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,11 +48,14 @@ def get_setup_version(reponame):
     import json, importlib
```
Comment (philippjfr) [inline setup.py]: This ``importlib`` import no longer appears to be used and is causing flake errors.

Comment (jlstevens) [reply setup.py]: Thanks. Must be a difference in flake8 versions as this wasn't caught in holoviews? I'll fix it in both places now.

Comment (ceball) [reply setup.py]: (it's a difference in how flake8 is run)

Comment (philippjfr) [reply setup.py]: Right, in holoviews it's only run on the holoviews and tests packages.

Comment (ceball) [reply setup.py]: When I inflict pyct on you, it will be the same everywhere, and everyone will hate me ;)

**Comment by jlstevens** [comment]
Looks like the tests will pass this time.

**Pull request merged**